proselint/checks/cliches/misc.py

```diff
@@ -850,4 +920,167 @@ def check_cliches_gnu_diction(text):
         "you are hereby advised that",
     ]
 
-    return existence_check(text, list, err, msg, join=True, ignore_case=True)
+    return existence_check(text, items, err, msg, ignore_case=True)
+
+
+def check_cliches_nigel(text: str) -> list[ResultCheck]:
```

Comment:
Is this not now the longest cliché check? Should we consider splitting this too for parallelization gains?

Reply:
you can run `scripts/benchmark_checks.py` - this is the output for me, running though demo.md:

- timings are for 10 iterations of all checks running in serial on my hardware

```
count:  147 checks
min:    6.172998109832406e-05 s
median: 0.0015084069455042481 s
mean:   0.011071045536921696 s
max:    0.12141467403853312 s
sum:    1.6274436939274892 s
```
the eight slowest checks + cliches_nigel:

```
proselint.checks.cliches.misc.check_cliches_nigel: 18.192 ms
[...]
proselint.checks.terms.venery.check: 69.674 ms
proselint.checks.spelling.misc.check: 77.394 ms
proselint.checks.misc.preferred_forms.check: 82.052 ms
proselint.checks.redundancy.misc.check_atd_2: 100.877 ms
proselint.checks.spelling.able_ible.check: 105.087 ms
proselint.checks.redundancy.misc.check_atd_1: 116.106 ms
proselint.checks.needless_variants.misc.check_1: 116.221 ms
proselint.checks.needless_variants.misc.check_2: 121.415 ms
```